assets/clients/clients.yaml

```
@@ -101,7 +101,7 @@ clients:
     official: false
     beta: true
     downloads:
-      - text: IzzyOnDroid
+      - text: izzisoft
```

Good catch, I think the text is a leftover. There is already an icon for izzisoft included.

```suggestion
      - icon: izzisoft
```